src/app/bindings.ts

```diff
@@ -0,0 +1,36 @@
+import {AppBinding, AppState, AppsState} from 'mattermost-redux/types/apps';
+
+class Bindings {
+    // getBindings returns bindings defined for all locations in the app
+    getBindings(): AppState {
```

Comment:
This seems to make more sense

```suggestion
    getBindings(): AppBinding[] {
```

Reply:
Agree. Thanks!